lib/moonshot/stack_template.rb

```diff
@@ -1,7 +1,5 @@
-require 'json'
-
 module Moonshot
-  # A StackTemplate loads the JSON template from disk and stores information
+  # A StackTemplate loads the template from disk and stores information
   # about it.
   class StackTemplate
```

Comment:
Make the base initialize raise and move whatever common work it does into a helper.

[NotImplementedError](https://ruby-doc.org/core-2.2.0/NotImplementedError.html) sounds kinda right here, but of course isn't. There's probably something appropriate-ish.